README.md

```diff
@@ -428,12 +428,24 @@ good
 
 bad
   .chain(eitherToMaybe) // Nothing
+
+// Transformations from array and back
+const listFromArray =
+  List.fromArray([1,2,3]) // List [ 1, 2, 3 ]
+
+const arrayFromList =
+  listToArray(listFromArray) // [ 1, 2, 3 ]
+
+const andBackAgain =
+  arrayToList(arrayFromList) // List [ 1, 2, 3 ]
 ```
 
 #### Transformation Signatures
 | Transform | ADT signature | Function Signature |
 |---|---|---|
+| `arrayToList` | `a -> List a` | `(a -> b) -> a -> b` |
 | `eitherToAsync` | `Either e a -> Async e a` | `(a -> Either e b) -> a -> Async e b` |
 | `eitherToMaybe` | `Either b a -> Maybe a` | `(a -> Either c b) -> a -> Maybe b` |
+| `listToArray` | `List a -> a` | `(a -> List b) -> a -> List b` |
```

Comment:
This should really be something like:
* ADT: `List a -> [ a ]`.
* Function: `(a -> List b) -> [ b ]`